docs/user-guide/logging/overview.md

```diff
@@ -0,0 +1,107 @@
+
+## Logging at the node level
+
+![Node level logging](/images/docs/user-guide/logging/logging-node-level.png)
+
+Everything a containerized application writes to `stdout` and `stderr` goes to a container runtime. Currently, in the Docker runtime, that output is written to a file using the standard [json logging driver](https://docs.docker.com/engine/admin/logging/overview).
+
+**Note:** The json logging driver treats each line as a separate message. There is no direct support for multi-line messages such as stack traces.
+
+By default, if a container restarts, kubelet keeps one terminated container with its logs. If a pod is evicted from the node, all corresponding containers are also evicted, along with their logs.
+
+An important consideration in node-level logging is implementing log rotation, so that logs don't consume all available storage on the node. Kubernetes uses the [`logrotate`](http://www.linuxcommand.org/man_pages/logrotate8.html) tool to implement log rotation.
+
+Kubernetes performs log rotation daily, or if the log file grows beyond 10MB in size. Each rotation belongs to a single container; if the container repeatedly fails or the pod is evicted, all previous rotations for the container are lost. By default, Kubernetes keeps up to five logging rotations per container.
+
+The Kubernetes logging configuration differs depending on the node type. For example, you can find detailed information for GCI in the corresponding [configure helper](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/cluster/gce/gci/configure-helper.sh#L96).
+
+When you run [`kubectl logs`](/docs/user-guide/kubectl/kubectl_logs), as in the basic logging example, the kubelet on the node handles the request and reads directly from the log file, returning the contents in the response. Note that `kubectl logs` **only returns the last rotation**; you must manually extract prior rotations, if desired.
+
+### System components logs
+
+Kubernetes system components use a different logging mechanism than the application containers in pods. Components such as `kube-proxy` (among others) use the [glog](https://godoc.org/github.com/golang/glog) logging library. You can find the conventions for logging severity for those components in the [development docs on logging](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/logging.md).
+
+System components write directly to log files in the `/var/log` directory in the node's host filesystem. Like container logs, system component logs are rotated daily and based on size. However, system component logs have a higher size retention: by default, they store 100MB.
+
+## Cluster-level logging architectures
+
+While Kubernetes does not provide a native solution for cluster-level logging, there are several common approaches you can consider:
+
+* You can use a node-level logging agent that runs on every node as a manifest pod or a DaemonSet replica.
```

Comment:
Let's do not mention about manifest pods here. This is a deprecated way to run per node agent